code/datums/mutations/sight.dm

```diff
@@ -40,6 +40,7 @@
 	quality = POSITIVE
 	difficulty = 18
 	text_gain_indication = "<span class='notice'>You can see the heat rising off of your skin...</span>"
+	text_lose_indication = "<span class='notice'>You can no longer see the heat rising off of your skin...</span>"
```

Comment:
u couldve done better

Reply:
i was worried about fixing the bug, didnt want to try to make anything fancy, outside the scope of the PR tbh. that and im not very creative for that kinda stuff

Reply:
yeah its whatever